File: Quiz.Task/src/Quiz.Task.Service/Models/Member.cs
using Quiz.Common.Models;

namespace Quiz.Task.Service.Models
{
    public class Member : IModel
    {
        public int Id { get; set;}
        public int UserId { get; set;}
        public int ClassId { get; set;}
        public virtual UserInfo User { get; set;}  
        public virtual Class Class { get; set;}
        
    }
}

File: quiz/lib/models/member.dart
import 'package:quiz/models/class.dart';
import 'package:quiz/models/user.dart';

class Member{
  int memberId;
  UserInfo user;
  Class e_class;
  
  Member({required this.memberId, required this.user,required this.e_class});

  factory Member.fromJson(Map<String, dynamic> json) => Member(
    memberId: json['id'],
    user: UserInfo.fromJson(json['member']),
    e_class: Class.fromJson(json['class'])
  );
}